packages/app/state/tokenList.ts

```diff
@@ -0,0 +1,10 @@
+import { type TokenDocument } from '@walless/store';
```

Comment:
we dont need this state fragment? we already have useTokens() hook and state associate to it.

Reply:
already removed fragment state and change it to `unknown`